Rewrite `fancyhdr` tutorial (#40)
oetiker/lshort: book/src/basics.tex

\begin{lscommand}
\ci{thispagestyle}\verb|{|\emph{style}\verb|}|
\end{lscommand}

> You may also control the style of the displayed page numbers. To change it use

Reviewer: ```suggestion
You can also control the style of the displayed page numbers. To change the style use
```

the
\begin{lscommand}
  \ci{pagenumbering}[style]
\end{lscommand}

> command, where \carg{style} is one of the styles presented in

Reviewer: ```suggestion
command, where \carg{style} is one of the styles listed in
```
